assets/css/styleguide.scss

```diff
@@ -858,3 +883,126 @@ code[class*="language-"], pre[class*="language-"] {
 .usa-code-sample pre[class*="language-"] {
   margin-top: 0;
 }
+
+.preview {
+  .usa-background-dark {
+    display: inline-block;
+    padding: {
+      left: 1em;
+      right: 1em;
+    }
+  }
+}
+
+.usa-link-visited {
```

Comment:
It brings into question how many color helpers we want to provide. For example, developers can add this background color in however they want. 

These are all the colored backgrounds we are showing as possible background colors to use:

![screen shot 2015-08-31 at 9 53 28 am](https://cloud.githubusercontent.com/assets/5249443/9584493/3865bb72-4fc6-11e5-84bb-21f05c91e50c.png)

The last thing we decided was to _not_ provide the color helpers.